frontend/app/src/composables/info/chains.ts

```diff
@@ -78,7 +78,29 @@ export const useSupportedChains = createSharedComposable(() => {
   const getChainInfoById = (
     chain: MaybeRef<Blockchain>
   ): ComputedRef<ChainInfo | null> =>
-    computed(() => get(supportedChains).find(x => x.id === get(chain)) || null);
+    computed(() => {
+      const chainVal = get(chain);
+      const found = get(supportedChains).find(x => x.id === chainVal);
+
+      if (!found) {
+        return null;
+      }
+
+      if (chainVal === Blockchain.ETH2) {
+        return {
+          ...found,
+          name: 'ETH Staking'
```

Comment:
> You mean that we return `Ethereum Beaconchain`? It comes from a data structure that we have named `SupportedBlockchain` that has `ETHEREUM_BEACONCHAIN` and serializes to `Ethereum Beaconchain`

Ok so it is implementation, do you think we can move the text exception to the backend in this case and returns `ETH Staking` instead of doing it in the frontend? it will save us some time in the future trying to figure out why it doesn't change :P

Reply:
@lukicenturi so let's remove this exception and the backend team will take care of it.

Reply:
Okay, reverted